### ISSUE TITLE
For CTEX, system PATH is not right

### ISSUE THREAD

**Issue by c1111f1** [SublimeText/LaTeXTools#1303]
For current ctex version 2.9.3, the installation file of ctex cannot build the right system path X:\CTEX\MiKTeX\miktex\bin. This will cause "check system" cannot find any compiler.

Could you please improve this function?

**Comment by r-stein** [comment]
Does it work if you manually add it to the PATH?

**Comment by c1111f1** [comment]
Yes. I know how to solve the problem. I am writing a latex tutorial and recommand my reader to use latextools. They asked me a lot about this issue. I told them how to fix this issue by adding the path. But if the tool can address this, it will be great. Thank you very much!


原始邮件
发件人:Richard Steinnotifications@github.com
收件人:SublimeText/LaTeXToolsLaTeXTools@noreply.github.com
抄送:c1111f1fc@oooole.net; Authorauthor@noreply.github.com
发送时间:2018年3月25日(周日) 23:20
主题:Re: [SublimeText/LaTeXTools] For CTEX, system PATH is not right(#1303)


Does it work if you manually add it to the PATH?

**Comment by r-stein** [comment]
We have [this setting](https://github.com/SublimeText/LaTeXTools/blob/st3-3.14.2/LaTeXTools.sublime-settings#L220) to adapt the path while executing our commands. Not sure if we can detect it automatically.

**Comment by deathaxe** [comment]
If required tools are not already available via global $PATH, the only viable option is to manually customize it or use `texpath` setting as already recommended.

It's not viable to do tons of guesswork to try to find tools which can be installed at arbitrary locations, especially on Windows.
